### ISSUE TITLE
Showing Comments Bug

### ISSUE THREAD

**Issue by osseonews** [ahmedkaludi/accelerated-mobile-pages#1435]
If you choose the setting in the plugin to show comments, comments are not displayed if comments are closed. This is probably an unintended feature because there are many blog posts that may get 100's of comments, and at some point we need to shut down the ability to add a comment. However, this doesn't mean that we don't want the comments to appear in AMP. Your code fails to display comments even when there are comments to display, if comments are closed. 

For an example, you can go here:
https://www.osseonews.com/controlling-pain-after-implant-placement/ (95 comments)
https://www.osseonews.com/controlling-pain-after-implant-placement/amp/ (0 comments displayed even though we have enabled comments in the AMP settings, and comments show on all our other pages, as long as comments are not disabled.

**Comment by osseonews** [comment]
BTW, to fix this I just commented out if (!comments_open() ||  in the comments.php file in the design I was using. This !comments_open() ||  is not necessary. If you don't want to show comments in the AMP theme, you just choose that setting in the admin panel. But, if you choose to show comments, and comments are now closed for various reasons, the comments still need to show up on the page. comment_open() is incorrect logic here.
